curriculum/challenges/english/14-responsive-web-design-22/learn-css-grid-by-building-a-magazine/step-024.md

```diff
@@ -205,9 +205,8 @@ assert(document.querySelectorAll('.image-wrapper img')?.[2]?.getAttribute('heigh
             height="400"
           />
           <blockquote class="image-quote"></blockquote>
-          <img />
 --fcc-editable-region--
-
+<img />
```

Comment:
```suggestion
          <img />
```
Nitpicking.

Reply:
I was wondering about that. I wasn't sure where in the editor the tag should line up. Often the lessons have the tag up against the left margin, even though in the HTML it is indented as a child element. Are you saying you want me to align it to be in line with where it was in the HTML?  Let me know if that is what you want and I'll change it right away.

Reply:
Please use the `Commit suggestion` button to accept @gikf 's suggestion.

Reply:
That's how I'd do it. It would be a bit weird for me, if it'd be now aligned to the left, but in both previous and next steps it would be indented with the rest of code.

Reply:
Ok, I did that. I won't be so quick to click on the Close issue button next time. I also noticed that I have "typos" in the branch name I created even though this is not a typo. Hopefully, that is not a big deal.